skl2onnx/operator_converters/linear_classifier.py

```diff
@@ -53,11 +59,11 @@ def convert_sklearn_linear_classifier(scope, operator, container):
                 is not SOFTMAX.
                 https://github.com/scikit-learn/scikit-learn/blob/bac89c253b35a8f1a3827389fbee0f5bebcbc985/sklearn/linear_model/base.py#L300
                 """ # noqa
-                classifier_attrs['post_transform'] = 'LOGISTIC'
+                classifier_attrs['post_transform'] = post_transform
```

Comment:
Why do we have three same statements with if-else? Can't we just remove all the if-else and just have the assignment statement?

Reply:
Fixed.